CinemaTicketSalesSystem/CinemaTicketSalesSystem/Controllers/HomeController.cs

```diff
@@ -10,14 +11,24 @@ namespace CinemaTicketSalesSystem.Controllers
 {
     public class HomeController : Controller
     {
-        ApplicationDbContext db = new ApplicationDbContext();
+        private ApplicationDbContext db = new ApplicationDbContext();
```

Comment:
Usually private keyword is not used. Just name your variable with '_' in the beginning, it is commonly used practice for private variables.
```
ApplicationDbContext _db = new ApplicationDbContext();
```